src/plugin/memory-webview-main.ts

```diff
@@ -231,28 +233,40 @@ export class MemoryWebview implements vscode.CustomReadonlyEditorProvider {
         return { title, bytesPerWord, wordsPerGroup, groupsPerRow, scrollingBehavior, visibleColumns, addressPadding, addressRadix, showRadixPrefix };
     }
 
-    protected async readMemory(request: DebugProtocol.ReadMemoryArguments): Promise<MemoryReadResult> {
+    protected async readMemory(request: ReadMemoryArguments): Promise<ReadMemoryResult> {
         try {
             return await this.memoryProvider.readMemory(request);
         } catch (err) {
-            outputChannelLogger.error('Error fetching memory', err instanceof Error ? `: ${err.message}\n${err.stack}` : '');
+            this.logError('Error fetching memory', err);
         }
     }
 
-    protected async writeMemory(request: DebugProtocol.WriteMemoryArguments): Promise<MemoryWriteResult> {
+    protected async writeMemory(request: WriteMemoryArguments): Promise<WriteMemoryResult> {
         try {
-            return await this.memoryProvider.writeMemory(request);
+            return this.memoryProvider.writeMemory(request);
```

Comment:
Inside a `try` block, the `await` is required; otherwise a rejection can't be caught here.

Reply:
Great catch!

Reply:
No pun intended? :-)

Reply:
Dang, I didn't even notice, I could have appeared so clever ;-)